xml/deploy_additional.xml

```diff
@@ -300,6 +300,53 @@
       </listitem>
      </itemizedlist>
     </note>
+    <important>
+     <title>Incorrectly reported image sizes and the <option>emulate_legacy_capacity</option> option</title>
+     <para>
+      A bug in older kernels caused that an RBD target reported an image size
+      one block (512 bytes) larger than it actually was. Therefore, the LUN
+      could write one additional block of the image. To keep the image data
+      consistent, new kernels support the old behavior by default unless you
+      set <option>emulate_legacy_capacity</option> attribute to 0 for a target.
+     </para>
+     <para>
+      We recommend disabling the old behavior for newly created targets. If you
+      plan to enable the <option>object-map</option> feature for RBD images,
+      disabling <option>emulate_legacy_capacity</option> is required. Use the
+      &gwcli; shell to disable it:
+     </para>
+<screen>
+&prompt.root;&gwcli;
+&prompt.gwcli;cd /disks/<replaceable>POOL</replaceable>/<replaceable>IMAGE</replaceable>
+&prompt.gwcli;reconfigure emulate_legacy_capacity 0
+</screen>
+     <tip>
+      <para>
+       You can also disable the <option>emulate_legacy_capacity</option> option
+       in the &dashboard; when creating or editing the RBD target.
+      </para>
+     </tip>
+     <para>
+      For already existing RBD targets that do not use the
+      <option>object-map</option> feature, you may continue to use the
+      <option>emulate_legacy_capacity</option> enabled. In such case you can
+      safely ignore related <literal>target_core_rbd: beyond EOD</literal>
+      kernel messages on corresponding gateway nodes.
+     </para>
+     <para>
+      If you want to disable <option>emulate_legacy_capacity</option> for
+      existing targets to avoid potential data corruption, we recommend
+      increasing the image size by on 512B to include the additional boundary
+      block. Resize the image when it is not used right before disabling the
+      emulation. To see the current image size in bytes, use the following
```

Comment:
```suggestion
      To avoid data corruption, <option>emulate_legacy_capacity</option> should only be disabled on existing iSCSI LUNs if the RBD image is grown by one 512-byte block at the same time. Disconnect all iSCSI initiators and disable the LUN or iSCSI target, grow the image using the procedure below, disable <option>emulate_legacy_capacity</option> and then finally re-enable the LUN or iSCSI target. To see the current image size in bytes, use the following
```